src/DynamoCoreWpf/Rendering/HelixRenderPackage.cs

```diff
@@ -38,7 +38,7 @@ public IRenderPackage CreateRenderPackage()
     /// A Helix-specifc IRenderPackage implementation.
     /// </summary>
     [Obsolete("Do not use! This will be moved to a new project in a future version of Dynamo.")]
-    public class HelixRenderPackage : IRenderPackage, ITransformable, IRenderLabels, IRenderPackageSupplement, IInstancingRenderPackage, IRenderInstancedLabels
+    internal class HelixRenderPackage : IRenderPackage, ITransformable, IRenderLabels, IRenderPackageSupplement, IInstancingRenderPackage, IRenderInstancedLabels
```

Comment:
I wonder if this will break Player? @twastvedt @saintentropy ? Even if we allow internals to be visible to player, it's kind of a recipe for breaks (since it's in another repo) if player has a dependency here I am not mis-remembering?

Reply:
We do refer to HelixRenderPackage. We also currently have internal access, so this should be ok for us (I think?). We'd just want to not let go of access for DynamoPlayer.Extension in [this PR](https://github.com/DynamoDS/Dynamo/pull/14552/files#diff-de50d688464d5b077d43844b2b81cedfd70504fa8c8591feb6c35cfe44f92582L43), but we were considering doing that anyway to be safe for the future.

Reply:
What I mean is - once this class is made internal, even if you have internal access, developers will be free to change any and all members of this class, or even delete the class.

If it's truly an API that you depend on we should abstract that out to some class, or internal interface we are okay to continue providing.

Reply:
🤦 I was on the wrong branch when I went searching for HelixRenderPackage in Player. We do **not** have any references to HelixRenderPackage in the current Player, we just make a HelixRenderPackageFactory.

Reply:
@sm6srw isn't that also a type you are making internal?

Reply:
Yes, I am leaning towards reverting all Helix changes.

Reply:
Reverted.